docker/test/integration/resources/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM elasticsearch:8.2.2
+COPY elasticsearch.yml /usr/share/elasticsearch/config/elasticsearch.yml
+COPY certs/elastic_cert.key /usr/share/elasticsearch/config/certs/elastic_cert.key
+COPY certs/elastic_cert.pem /usr/share/elasticsearch/config/certs/elastic_cert.pem
+COPY certs/elastic_transport.key /usr/share/elasticsearch/config/certs/elastic_transport.key
+COPY certs/elastic_transport.pem /usr/share/elasticsearch/config/certs/elastic_transport.pem
```

Comment:
Both the cert and the private keys are in PEM format. Maybe using the .key.pem and .crt.pem, or .key and .crt extension would be less confusing.

Reply:
good idea, I've changed them in https://github.com/apache/nifi-minifi-cpp/pull/1349/commits/fdb1bf2a34423815cfe2cb395e83dcee3bc2f1cd#diff-a10da10e9ba31d953619871d6557f4fe8bffa0b1ceb54bf8612b6de6bc1f33e1L3-L7